src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -81,7 +82,7 @@ private string GetProjectOrSolution()
                 //If a single solution is found, use it.
                 else if (possibleSolutionPath.Count() == 1)
                 {
-                    return possibleSolutionPath[0];
+                    return possibleSolutionPath.Single();
```

Comment:
[0] is faster, I imagine

Reply:
`.Single` guarantees only 1 entry (throws if count > 1). `[0]` doesn't. (Note the comment above the if-statement)

Reply:
Yes...but the if statement is pretty clear, so I don't think that's really necessary